src/main/java/com/losthex/App.java

```diff
@@ -0,0 +1,18 @@
+package com.losthex;
+
+import com.losthex.licensespring.MyLicenseManager;
+
+public class App {
+    public static void main(String[] args) {
+        System.out.println("Starting...");
+        MyLicenseManager licenseManagerClass = new MyLicenseManager();
+
+        if (args.length != 5  && args.length!=4) {
+            System.out.println("Four arguments required: apiKey, sharedKey, licenceKey, productName and optional isStaging(yes/no) variable");
```

Comment:
throw an exception here after logging it (invalid arguments exception or something similar)